README.md

```diff
@@ -115,6 +115,8 @@ The following sources are currently supported:
 
 - Itch ([itch.io](https://itch.io) app)
   - None of these entries have box art or icons, as they are specified as URLs rather than local files.
+  - All itch.io games are invoked directly through a discovered interpreter script
```

Comment:
This part is technically incorrect, since all of my entries (random couple games I installed) were just pointing directly to binaries with no interpreter specified. Can probably just remove the line.